fedimint-server-core/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "fedimint-server-core"
+version = { workspace = true }
+authors = { workspace = true }
+description = { workspace = true }
+documentation = { workspace = true }
+edition = { workspace = true }
+readme = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+keywords = { workspace = true }
+
+[lib]
+name = "fedimint_server_core"
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = { workspace = true }
+async-trait = { workspace = true }
+fedimint-api-client = { workspace = true }
```

Comment:
:sus: this one

Reply:
![image](https://github.com/user-attachments/assets/ca84c8d3-3a97-42fb-abf0-3ea05348ece4)


Ha, ha! I'm faster than CI.

Reply:
Probably a rebase artifact from reordering commits, I made this change to add that dep later, but shouldn't be in this commit yet indeed.